app/tasks.py

```diff
@@ -404,41 +406,33 @@ def get(self):
                 logging.info("Timeout occurs in datastore")
                 self.add_task_for_repo(
                     self.repo, self.task_name(), self.ACTION)
-            finally:
-                self._notify(count_of_unreviewed_notes)
 
-    def _notify(self, count_of_unreviewed_notes):
+    def _maybe_notify(self, count_of_unreviewed_notes):
         # TODO: Response should be modified
-        if self._is_ok_to_notify(count_of_unreviewed_notes):
-            self.send_mail(self._get_admin_user_email(),
+        if self._should_modify(count_of_unreviewed_notes):
+            self.send_mail(self._get_unreviewed_notes_email(),
                            subject="PersonFinder: Please review your notes",
```

Comment:
Optional: You may plan to change it later, but the response should contain the repository name at least.